designs/NoSQL_note.md

```diff
@@ -0,0 +1,83 @@
+# NoSQL
+
+Due to the technical limitations of setting up SQL, we decide to take the NoSQL approach by utilizing DynamoDB table on AWS, which has a trivial entry.
+
+## Tables
+
+### Customers:
+ - id: `string`
```

Comment:
Do we also want to consider names? That might be different from the id, which can be just alphanumerical gibberish.